onager/utils.py

```diff
@@ -1,9 +1,23 @@
 import csv
 from itertools import count, groupby
 import json
+import os
 
 from .constants import default_index
 
+def cpu_count():
+    # os.cpu_count()
+    #     returns number of cores on machine
+    # os.sched_getaffinity(pid)
+    #     returns set of cores on which process is allowed to run
+    #     if pid=0, results are for current process
+    #
+    # if os.sched_getaffinity doesn't exist, just return cpu_count and hope for the best
+    try:
+        return len(os.sched_getaffinity(0))
+    except AttributeError:
```

Comment:
I'm not sure this breaks by raising AttributeError on OSs that don't have this function. Are you sure about this? The docs don't mention anything.